docs/tutorials/lst_analysis_bootcamp_2018.ipynb

```diff
@@ -1233,7 +1243,7 @@
    "name": "python",
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
-   "version": "3.7.8"
+   "version": "3.7.7"
```

Comment:
You are not the newest python version btw :)

Reply:
did I forget to strip the output again?

Reply:
No, that's always included, no worry.